okta_openvpn.py

```diff
@@ -84,7 +84,7 @@ def _validate_conn(self, conn):
         public_key_sha256 = hashlib.sha256(public_key_raw).digest()
         public_key_sha256_base64 = base64.b64encode(public_key_sha256)
 
-        if public_key_sha256_base64 not in self.pinset:
```

Comment:
This is necessary because pinsets are strings, and `public_key_sha256_base64` is bytes in the new python version.